convex/fileStorageHttp.ts

```
@@ -0,0 +1,237 @@
+    const file = formData.get('file') as File;
+    const fileMetadata = formData.get('metadata') as string;
+
+    if (!file) {
+      return createErrorResponse('No file provided', 400);
+    }
+
+    if (!fileMetadata) {
+      return createErrorResponse('No file metadata provided', 400);
+    }
+
+    let parsedMetadata;
```

Biome (1.9.4)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)